packages/react-styles/src/css/components/Topology/topology-pipelines.css

```diff
@@ -0,0 +1,352 @@
+  --pf-topology-pipelines__pill__background--Fill: var(--pf-topology-pipelines__pill--m-warning--Background--Fill);
+  --pf-topology-pipelines__pill__background--Stroke: var(--pf-topology-pipelines__pill--m-warning--Background--Fill);
+}
+
+.pf-topology-pipelines__pill.pf-m-selected.pf-m-danger .pf-topology-pipelines__pill--background {
+  --pf-topology-pipelines__pill__background--Fill: var(--pf-topology-pipelines__pill--m-danger--Background--Fill);
+  --pf-topology-pipelines__pill__background--Stroke: var(--pf-topology-pipelines__pill--m-danger--Background--Fill);
+}
+
+/* Node label */
+.pf-topology-pipelines__pill__text {
+  font-size: var(--pf-topology-pipelines__pill__text--FontSize);
+  font-family: var(--pf-topology-pipelines__pill__text--FontFamily);
+  fill: var(--pf-topology-pipelines__pill--Color);
+}
+
+.pf-topology-pipelines__pill.pf-m-selected .pf-topology-pipelines__pill__text {
+  --pf-topology-pipelines__pill--Color: var(--pf-topology-pipelines__pill--m-selected--text-Color);
+}
+
+.pf-topology-pipelines__pill.pf-m-selected.pf-m-info .pf-topology-pipelines__pill__text {
+  --pf-topology-pipelines__pill--Color: var(--pf-topology-pipelines__pill--m-selected--m-info--text-Color);
+}
+
+.pf-topology-pipelines__pill.pf-m-selected.pf-m-success .pf-topology-pipelines__pill__text {
+  --pf-topology-pipelines__pill--Color: var(--pf-topology-pipelines__pill--m-selected--m-success--text-Color);
+}
+
+.pf-topology-pipelines__pill.pf-m-selected.pf-m-warning .pf-topology-pipelines__pill__text {
+  --pf-topology-pipelines__pill--Color: var(--pf-topology-pipelines__pill--m-selected--m-warning--text-Color);
+}
+
+.pf-topology-pipelines__pill.pf-m-selected.pf-m-danger .pf-topology-pipelines__pill__text {
+  --pf-topology-pipelines__pill--Color: var(--pf-topology-pipelines__pill--m-selected--m-danger--text-Color);
+}
+.pf-topology-pipelines__pill.pf-m-selected.pf-m-success .pf-topology-pipelines__pill__text {
+  --pf-topology-pipelines__pill--Color: var(--pf-topology-pipelines__pill--m-selected--m-skipped--text-Color);
+}
+.pf-topology-pipelines__pill.pf-m-selected.pf-m-running .pf-topology-pipelines__pill__text {
+  --pf-topology-pipelines__pill--Color: var(--pf-topology-pipelines__pill--m-selected--m-running--text-Color);
+}
+.pf-topology-pipelines__pill.pf-m-selected.pf-m-skipped .pf-topology-pipelines__pill__text {
+  --pf-topology-pipelines__pill--Color: var(--pf-topology-pipelines__pill--m-selected--m-skipped--text-Color);
+}
+.pf-topology-pipelines__pill.pf-m-selected.pf-m-canceled .pf-topology-pipelines__pill__text {
+  --pf-topology-pipelines__pill--Color: var(--pf-topology-pipelines__pill--m-selected--m-canceled--text-Color);
+}
+.pf-topology-pipelines__pill.pf-m-selected.pf-m-pending .pf-topology-pipelines__pill__text {
+  --pf-topology-pipelines__pill--Color: var(--pf-topology-pipelines__pill--m-selected--m-running--text-Color);
+}
+
+.pf-topology-pipelines__pill--status {
```

Comment:
Here, either `.pf-topology-pipelines__pill-status` or `.pf-topology-pipelines__pill__status` depending on what you change from the note above.